data/category-ru-gov

```diff
@@ -0,0 +1,17 @@
+# Federal Gov.
+government.ru
+gov.ru
+
+# Federal Public Services
+gosuslugi.ru
+gu-st.ru
+
+# Moscow Public Services
+mos.ru
+
+# Moscow Region (Province) Public Services
+mosreg.ru
```

Comment:
```suggestion
# Public Services
mos.ru     #Moscow 
mosreg.ru  #Moscow Region (Province)
spb.ru     #St Petersburg
```

Would this be better? Humm

Reply:
Actually I could add all domains related to public services in Russia then